src/v2/mod.rs

```diff
@@ -1388,8 +1390,10 @@ impl Client {
     }
 
     /// Get information about the passive delegators at the end of a given
-    /// block. If the block does not exist [`QueryError::NotFound`] is
-    /// returned.
+    /// block.
+    /// If the block does not exist [`QueryError::NotFound`] is returned.
+    /// If the block is baked prior to protocol version 4,
+    /// [`QueryError::NotFound`] is returned.
```

Comment:
It would be better to phrase this as 
```suggestion
    /// If the block does not exist or is prior to protocol version 4 then [`QueryError::NotFound`] is returned.
```

Reply:
And the same for errors bellow.

Reply:
The same case can't be made for all of them, though. I found that only `get_passive_delegation_info` and `get_pool_info` returned `NotFound`. The rest returned `RPCError` (with the message "Invalid argument"). I'll try to phrase them as a single sentence anyway.

Reply:
The comment was about the phrasing, not the specific message.